src/docs/base.mdx

```diff
@@ -0,0 +1,174 @@
+    | <Copy value="0x4200000000000000000000000000000000000015" label="0x420...15" /> | <Reference label="L1Block" url="https://github.com/ethereum-optimism/optimism/blob/master/packages/contracts-bedrock/contracts/L2/L1Block.sol" /> | Responsible for maintaining L1 context in L2. This allows for L1 state to be accessed in L2 |
+    | <Copy value="0x4200000000000000000000000000000000000016" label="0x420...16" /> | <Reference label="L2ToL1MessagePasser" url="https://github.com/ethereum-optimism/optimism/blob/master/packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol" /> | Stores commitments to withdrawal transactions. When a user is submitting the withdrawing transaction on L1, they provide a proof that the transaction that they withdrew on L2 is in the `sentMessages` mapping of this contract |
+    | <Copy value="0x4200000000000000000000000000000000000017" label="0x420...017" /> | <Reference label="OptimismMintableERC721Factory" url="https://github.com/ethereum-optimism/optimism/blob/master/packages/contracts-bedrock/contracts/universal/OptimismMintableERC721Factory.sol" /> | Responsible for creating ERC721 contracts on L2 that can be used for depositing native L1 NFTs into |
+    | <Copy value="0x4200000000000000000000000000000000000018" label="0x420...018" /> | <Reference label="ProxyAdmin" url="https://github.com/ethereum-optimism/optimism/blob/master/packages/contracts-bedrock/contracts/universal/ProxyAdmin.sol" /> | Owner of all of the proxy contracts set at the predeploys. It is itself behind a proxy. The owner of the ProxyAdmin will have the ability to upgrade any of the other predeploy contracts |
+    | <Copy value="0x4200000000000000000000000000000000000019" label="0x420...019" /> | <Reference label="BaseFeeVault" url="https://github.com/ethereum-optimism/optimism/blob/master/packages/contracts-bedrock/contracts/L2/BaseFeeVault.sol" /> | Receives the basefees on L2. The basefee is not burnt on L2 like it is on L1. Once the contract has received a certain amount of fees, the ETH can be withdrawn to an immutable address on L1 |
+    | <Copy value="0x420000000000000000000000000000000000001a" label="0x420...01a" /> | <Reference label="L1FeeVault" url="https://github.com/ethereum-optimism/optimism/blob/master/packages/contracts-bedrock/contracts/L2/L1FeeVault.sol" /> | Receives the L1 portion of the transaction fees. Once the contract has received a certain amount of fees, the ETH can be withdrawn to an immutable address on L1 |
+
+</Section>
+
+<Section title="RPC-API">
+
+    <Legend />
+
+    The following endpoints behave differently compared to the canonical Ethereum L1 implementation of the JSON RPC API.
```

Comment:
Does base have the same RPC changes as Optimism? If that's so, then we would need to add them in the list here, f.e `optimism_outputAtBlock` etc.

And it seems that there is a new RPC endpoints - `optimism_rollupConfig` and `optimism_version`. Lets add them to the Optimism RPC data

Reply:
I tried calling each one of them using the Base rpc url, however each one of them returned "not supported" message. I removed them because the structure of the document implies we want to show only differences to mainnet and not to Optimism